webpack.config.js

```diff
@@ -202,15 +213,15 @@ module.exports = ({ production } = {}, { extractCss, analyze, tests, hmr, port,
   },
   performance: { hints: false },
   devServer: {
-    contentBase: outDir,
+    static: outDir,
```

Comment:
webpack-cli error pointed me to change to `static`.
Don't have the exact link. 
Let me know if you want that, so I can search for it